_posts/2016-08-06-release-add-ons-1.9.md

```diff
@@ -0,0 +1,37 @@
+---
+title:  "Release: Add-Ons 1.9"
+---
+
+We are proud to announce the availability of Add-On Administration 1.9, Add-On Self Administration 1.9,
+and Add-On Self Administration Plug-In API 1.6.
+These releases are mainly fixing bugs, but there's also a new feature in the Add-On Administration.
```

Comment:
Maybe better: "These releases are mainly fixing bugs. but there is one new feature in the Add-On Administration" ?